checkov/dockerfile/checks/ExposePort22.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
     def scan_entity_conf(self, conf):
         i = 0
         for expose_term in conf:
-            if "22" in expose_term['value'].split(' '):
+            if "22" | "22/tcp" in expose_term['value'].split(' '):
```

Comment:
this syntax doesn't do what you think it should do 🙂 
```suggestion
            if any(port in expose_term['value'].split(' ') for port in ("22", "22/tcp")):
```
you need to do something like that

Reply:
Thanks for the review I will rectify tomorrow I can now why that wouldn't work